scico/functional/_denoiser.py

```diff
@@ -110,7 +111,7 @@ def prox(self, x: JaxArray, lam: float = 1.0, **kwargs) -> JaxArray:
         # undo squeezing, if neccessary
         y = y.reshape(x_in_shape)
 
-        return y
+        return jax.device_put(y)
```

Comment:
The return type here was a numpy ndarray? I would have expected `hcb.call` to keep things within the jax world.

Reply:
You are right, the device_put wasn't necessary here. Fixed it.